sandbox/monorepo/generate/index.js

```diff
@@ -0,0 +1 @@
+console.log('Hello, world!');
```

Comment:
This **should** be annotated with an error as `noConsole` is enabled in the root config.